qsvt wire order (#6212)
PennyLaneAI/pennylane: doc/releases/changelog-dev.md

* Fix `qml.PrepSelPrep` template to work with `torch`:
  [(#6191)](https://github.com/PennyLaneAI/pennylane/pull/6191)

> * The wires of the ``qml.QSVT`` template take first the ``projector`` wires and then the ``UA`` wires following the expected order of the decomposition.
>   [(#6212)](https://github.com/PennyLaneAI/pennylane/pull/6212)

Reviewer: ```suggestion
* The ``qml.QSVT`` template now order the ``projector`` wires first and the ``UA`` wires second, which is the expected order of the decomposition.
  [(#6212)](https://github.com/PennyLaneAI/pennylane/pull/6212)
```
